yt/frontends/sdf/data_structures.py

```diff
@@ -119,6 +118,8 @@ def _parse_parameter_file(self):
             self._periodicity = (True, True, True)
         else:
             self._periodicity = (False, False, False)
+        if self._domain_override:
+            self._disable_periodicity_for_bbox()
```

Comment:
I think that this will wrongfully emit a warning even in case the `else` right before this is hit. IMO this conditional should be included in the previous one.

Reply:
This is fixed in the latest commit--thanks for pointing it out